dataqs/helpers.py

```diff
@@ -381,3 +381,21 @@ def __init__(self, content):
 
     def raise_for_status(self):
         pass
+
+
+def add_keywords(keyword_list, extra_keywords):
+    """Adds extra_keywords list to the keyword_list"""
+
+    # check if datetime and category already exist in the
+    # keyword list
+
+    keywords_added = [k for k in keyword_list
+                      if k.startswith('category:') or k.startswith('datetime:')]
+
+    if keywords_added:
+        filtered_keywords = [k for k in keyword_list
+                             if not k.startswith('category:') or
+                             not k.startswith('datetime:')]
```

Comment:
I thik I got it this time. https://github.com/OpenGeoscience/dataqs/pull/48/commits/17a5d0eb1a674f5bdbe5e59daac893ac3ac6cf72